openedx/features/edly/utils.py

```diff
@@ -25,20 +25,24 @@ def user_has_edly_organization_access(request):
     Returns:
         bool: Returns True if User has Edly Organization Access Otherwise False.
     """
-    if not request.user.edly_profile:
-        return False
 
-    current_site = request.site
-    edly_sub_org = EdlySubOrganization.objects.filter(
-        Q(lms_site=current_site) |
-        Q(studio_site=current_site)
-    ).first()
+    try:
+        if not request.user.edly_profile:
+            return False
+
+        current_site = request.site
+        edly_sub_org = EdlySubOrganization.objects.filter(
+            Q(lms_site=current_site) |
+            Q(studio_site=current_site)
+        ).first()
```

Comment:
This can be done as
```
edly_sub_org = EdlySubOrganization.objects.get(
    Q(lms_site=current_site) |
    Q(studio_site=current_site)
)
```
with exception `DoesNotExist`

Reply:
Feedback Implemented